public_html/wp-content/plugins/wordcamp-organizer-survey/includes/debrief-survey/email.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Creates the Organizer Debrief Survey email in the Camptix email queue.
+ */
+
+namespace WordCamp\OrganizerSurvey\DebriefSurvey\Email;
+
+defined( 'WPINC' ) || die();
+
+/**
+ * Constants.
+ */
+const EMAIL_KEY_ID = 'organizer_debrief_survey_email';
+
+/**
+ * Return the email ID.
+ *
+ * @return mixed
+ */
+function get_email_id() {
+	return get_option( EMAIL_KEY_ID );
+}
+
+/**
+ * Return lead organizer email.
+ *
+ *  @return string
+ */
+function get_lead_organizer_email() {
+	$meta = get_wordcamp_post()->meta;
+	return $meta['Email Address'][0];
+}
+
+/**
+ * Return lead organizer full name.
+ *
+ * @return string
+ */
+function get_lead_organizer_full_name() {
+	$meta = get_wordcamp_post()->meta;
+	return $meta['Organizer Name'][0];
+}
+
+/**
+ * Return WordCamp location.
+ *
+ * @return string
+ */
+function get_wordcamp_location() {
+	$meta = get_wordcamp_post()->meta;
+	return $meta['Location'][0];
+}
+
+
+/**
+ * Return a token to put into the survey URL.
+ *
+ * @return string
+ */
+function get_encryption_token() {
+	$wordcamp_id = get_current_blog_id();
+	return hash_hmac( 'sha1', $wordcamp_id, ORGANIZER_SURVEY_ACCESS_TOKEN_KEY );
+}
+
+/**
+ * Returns content for the reminder email.
+ *
+ * @return string
+ */
+function get_email_content() {
+	$wordcamp_name   = get_wordcamp_name();
+	$survey_page_url = 'https://central.wordcamp.test/organizer-survey-event-debrief/?t=' . get_encryption_token()
```

Comment:
```suggestion
	$tld = get_top_level_domain();
	$survey_page_url = "https://central.wordcamp.$tld/organizer-survey-event-debrief/?t=" . get_encryption_token()
```

Reply:
Ah wasn't aware there's already a function like this. Was planning to handle that before shipping to production. 👍